jujugui/static/gui/src/app/components/entity-details/content/resources/resources.js

```diff
@@ -23,39 +23,77 @@ YUI.add('entity-resources', function() {
   juju.components.EntityResources = React.createClass({
 
     propTypes: {
+      apiUrl: React.PropTypes.string.isRequired,
+      entityId: React.PropTypes.string.isRequired,
       pluralize: React.PropTypes.func.isRequired,
       resources: React.PropTypes.array
     },
 
+    getInitialState: function() {
+      const props = this.props;
+      return {
+        baseUrl: `${props.apiUrl}/${props.entityId}/resource`
```

Comment:
Should this use `jujulib.URL.fromLegacyString(props.entityId).legacyPath()` instead? That way we are being more explicit about what we are expecting and also we avoid including the cs: schema in the resulting URL. But maybe I am missing something and the cs: schema is never there.